python/dgl/distributed/graph_partition_book.py

```diff
@@ -923,6 +982,34 @@ def etypes(self):
         """
         return self._etypes
 
+    @property
+    def canonical_etypes(self):
+        """Get the list of canonical edge types
```

Comment:
Missing `Returns` section. Please add a note about what happens if the keys of `edge_map` and `etypes` are string.

Reply:
added...